hw1/01-fizzbuzz.js

```diff
@@ -10,7 +10,21 @@ Use console.log() to write the proper output to the command line.
 **/
 
 const fizzbuzz = () => {
-  // Add your code here
+  for (let i = 0; i < 100; i++) {
+    curr = (i + 1)
```

Comment:
Why do curr = (i+1), when you can run the loop from 1 to 100 directly, it increases the computation

Reply:
Thank you for the feedback! I have implemented both suggestions. Please approve when you get the chance.